### PR TITLE
Improve My Account app as per suggestions

### PR DESCRIPTION
**Pull request by thivi**

### CODE REVIEW

File: apps/myaccount/src/components/change-password/change-password.tsx [hunk 1]
```diff
@@ -19,12 +19,18 @@
...
 import { updatePassword } from "../../api";
 import { SettingsSectionIcons } from "../../configs";
+import { CommonConstants } from "../../constants";
 import { AlertInterface, AlertLevels } from "../../models";
+import { AppState } from "../../store";
+import { setOpenAction } from "../../store/actions";
 import { endUserSession } from "../../utils";
 import { EditSection, SettingsSection } from "../shared";
+
+
```
Comment (NipuniBhagya) [inline apps/myaccount/src/components/change-password/change-password.tsx:33]:
```suggestion
```

File: apps/myaccount/src/models/global.ts [hunk 2]
```diff
@@ -26,4 +26,5 @@ export interface GlobalInterface {
     isApplicationsPageVisible: boolean;
     isGlobalLoaderVisible: boolean;
     supportedI18nLanguages: SupportedLanguagesMeta;
+    openAction: string;
```
Comment (NipuniBhagya) [inline apps/myaccount/src/models/global.ts:29]: Should we rethink the term openAction? I feel like this term is a bit misleading given that we use it to decide the currently editing section.

File: apps/myaccount/src/store/actions/global.ts [hunk 3]
```diff
@@ -88,4 +89,14 @@ export const setSupportedI18nLanguages = (
...
     payload: languages,
     type: GlobalActionTypes.SET_SUPPORTED_I18N_LANGUAGES
+    });
+
+/**
+ * Redux action to set an action to open.
+ *
+ * @param {string} action - The action to set to open.
+ */
+export const setOpenAction = (action: string): SetOpenAction => ({
```
Comment (NipuniBhagya) [inline apps/myaccount/src/store/actions/global.ts:99]: Better if we can rethink the name

**Review by NipuniBhagya — APPROVED** [review]

File: apps/myaccount/src/components/change-password/change-password.tsx [hunk 4]
```diff
@@ -19,12 +19,18 @@
 import { Field, Forms, Validation, useTrigger } from "@wso2is/forms";
 import React, { FunctionComponent, useState } from "react";
 import { useTranslation } from "react-i18next";
+import { useDispatch, useSelector }from "react-redux";
```
Comment (NipuniBhagya) [inline apps/myaccount/src/components/change-password/change-password.tsx:22]:
```suggestion
import { useDispatch, useSelector } from "react-redux";
```

File: apps/myaccount/src/models/global.ts (hunk 2 above)
Comment (thivi) [reply apps/myaccount/src/models/global.ts]: Fixed by 26e3ed87

File: apps/myaccount/src/store/actions/global.ts (hunk 3 above)
Comment (thivi) [reply apps/myaccount/src/store/actions/global.ts]: Fixed by 26e3ed87.
`setOpenAction` has been replaced with `setActiveForm`.

**Pull request merged**